tests/integration/utils/cypress-commands.ts

```diff
@@ -30,52 +29,60 @@ Cypress.Commands.add('createLandRequest', (title: string, isSuccessful: boolean)
     body = { ...body, ['fail.txt']: '' };
   }
 
-  cy.request({
-    method: 'POST',
-    url: `https://api.bitbucket.org/2.0/repositories/${username}/${repo}/src`,
-    body,
-    auth: {
-      username,
-      password,
-    },
-    form: true,
-  });
-
-  cy.request({
-    method: 'POST',
-    url: `https://api.bitbucket.org/2.0/repositories/${username}/${repo}/pullrequests`,
-    body: {
-      title: title,
-      source: {
-        branch: {
-          name: title,
-        },
-      },
-      destination: {
-        branch: {
-          name: 'master',
-        },
+  try {
```

Comment:
wrapping into a try catch if not errors were swallowed

Reply:
The diff is super cooked here, was the only change moving this into the try catch?